api_app/analyzers_manager/file_analyzers/doc_info.py

```diff
@@ -147,8 +157,35 @@ def run(self):
             MimeTypes.ZIP2.value,
         ]:
             results["follina"] = self.analyze_for_follina_cve()
+            results["uris"].extend(self.get_docx_urls())
+
+        results["extracted_CVEs"] = self.analyze_for_cve()
+        results["uris"].extend(self.get_external_relationships())
+        results["uris"].extend(self.extract_urls_from_IOCs())
+
+        results["uris"] = list(set(results["uris"]))  # uniq
+
```

Comment:
it is better if you generically "try except" each these ones cause a single error from them would make all the analysis fail while, on the contrary, we would like to have the analysis with "success" and the correct "warning" errors. 
So, I would move the "try except" inside the single items and put them outside here.
That would also benefit unit testing. In that way you can test single methods more granurarly (for instance, you could also tests when they fail for specific reasons)